src/app/pages/card-import/card-import.page.ts

```diff
@@ -55,11 +55,15 @@ export class CardImportPage implements OnInit {
 
     if (this.id) {
       console.log('Current card ID: ', this.id);
-      this.cardService.getCard(this.id).subscribe(card => {
-        console.log("Card: ", card);
-        this.card = card;
-        this.frontImg = this.card.frontImg;
-        this.backImg = this.card.backImg;
+      this.cardService.getCard(this.id, (error, data) => {
```

Comment:
Looks good. Only I got an error for the `exchange.page.ts` file in relation to the `getCard()` function in the card service . Turns out you just missed adding the new callback argument syntax for this same code block in `exchange.page.ts`. It's a simple fix.